content/getting_started/UsingEnzyme.md

```diff
@@ -5,6 +5,19 @@ draft: false
 weight: 10
 ---
 
+## Differentiating C/C++
+
+Enzyme supports differentiating C/C++ code through ClangEnzyme and LLDEnzyme as compiler and linker plugins, respectively.
+Clang gives our plugin more flexibility in adding and ordering optimization passes than LLD and therefore using ClangEnzyme could result in better performance than LLDEnzyme.
+However, ClangEnzyme can only differentiate one compilation unit at a time and will therefore fail if the function which you try to differentiate calls functions
+in other compilation units (generally other .c or .cpp files). In these cases we recommend the use of LLDEnzyme in combination with LTO.
+Finally, for full control you can also compile your c/c++ project down to LLVM-IR and pass it directly to LLVMEnzyme, see the example below.
+
+An example for using LLDEnzyme as part of CMake is available [here](https://github.com/EnzymeAD/CMake-Template/blob/main/CMakeLists.txt).
```

Comment:
We should probably deprecate the template and instead point people to the example in Enzyme that is tested as part of CI: https://github.com/EnzymeAD/Enzyme/tree/main/enzyme/test/test_find_package